wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -756,7 +756,7 @@ def locales():
 
 @register.simple_tag()
 def slim_sidebar_enabled():
-    return 'slim-sidebar' in getattr(settings, 'WAGTAIL_EXPERIMENTAL_FEATURES', [])
```

Comment:
This should get its own 'proper' setting now (`WAGTAIL_SLIM_SIDEBAR = FALSE`?), if we're not  considering the slim sidebar to be experimental any more.

Reply:
👍  yep, will switch over to a separate setting.